### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by quantrpeter** [Nicholas3388/LuaNode#44]
Does this support Arduino? thx

**Comment by clearc352** [comment]
Arduino are support on my esp32

**Comment by MiManchiL** [comment]
> Arduino are support on my esp32

What Arduino development board should I choose? What are the parameter configurations? Could you please help me with that? thx

**Comment by Runogrog** [comment]
Yes, this project can support Arduino, but you’ll need to ensure that the library dependencies are compatible with the Arduino environment. Additionally, check the documentation for any specific configurations or settings that might be required for integrating the software with Arduino boards. If you encounter any issues, providing details about your setup will help in diagnosing the problem effectively.
